### DISCUSSION TITLE
Where is the lotus built-in marketplace?

### DISCUSSION THREAD

**Discussion by PatrickAlphaC** [filecoin-project/community#602] [Developers]
If I'm looking to make a deal with other storage providers, do I have to use a trusted 3rd party with Filecoin node ops?

**Comment by web3jenks** [comment]
Hey @PatrickAlphaC, how much data are we talking about here for your deal? The solution will vary quite a bit depending on your data sizes. 

Our data onboarding site has some information about onboarding data: https://dataonboarding.filecoin.io/

**Reply by PatrickAlphaC** [reply]
Just more curious about the data onboarding journey of a user. It looks like all the recommended marketplaces require going to a centralized website.

**Reply by web3jenks** [reply]
Yes, they are currently quickest ways to onboard data. 

To do it the decentralised way, a few tools can help: 
1. use [singularity](https://github.com/tech-greedy/singularity) project  ( for very large data sets) 
2. use [estuary](https://github.com/application-research/estuary) (self installed version)
3. upload data via a IPFS node, then use [lotus](https://lotus.filecoin.io/tutorials/lotus/store-and-retrieve/store-data/) or [venus node](url) to initiate a deal offer
